frontend/public/components/namespace.jsx

```diff
@@ -164,17 +164,16 @@ class PullSecret extends SafetyFirst {
 
 export const NamespaceLineCharts = ({ns}) => <div className="row">
   <div className="col-sm-6 col-xs-12">
-    <Line title="CPU Usage" query={[
+    <Line title="CPU Usage" namespace={ns.metadata.name} query={[
       {
         name: 'Used',
         query: `namespace:container_cpu_usage:sum{namespace='${ns.metadata.name}'}`,
       },
     ]} />
   </div>
   <div className="col-sm-6 col-xs-12">
-    <Line title="Memory Usage" query={[
-      {
-        name: 'Used',
+    <Line title="Memory Usage" namespace={ns.metadata.name} query={[
+      { name: 'Used',
```

Comment:
Nit: Was this newline accidentally removed?

Reply:
Thanks, fixed